### PR TITLE
Add all-ai server docker image

### PR DESCRIPTION
**Pull request by JamesXNelson**

### CODE REVIEW

**Review by niloc132 — COMMENTED** [review]
At a glance this looks good, but I'm not familiar in this area.

**Comment by JamesXNelson** [comment]
assigning to devin for merge.

**Comment by JamesXNelson** [comment]
Not sure where we document "here are the different server images you can use", but we'll want to add `all-ai` to that list.

File: .github/workflows/build-ci.yml [hunk 1]
```diff
@@ -220,6 +234,18 @@ jobs:
...
           push: ${{ github.event_name != 'pull_request' }}
 
+      - name: Docker build server all-ai
+        uses: docker/build-push-action@v2
+        with:
+          build-args: |
+            BASE=deephaven/all-ai-base:local-build
+            SERVER=server-all-ai
+            DEEPHAVEN_VERSION=${{ steps.deephaven_version.outputs.deephaven_version }}
+          tags: ${{ steps.docker_meta_server_tensorflow.outputs.tags }}
```
Comment (devinrsmith) [inline .github/workflows/build-ci.yml:244]: Wrong tags

**Review by devinrsmith — CHANGES_REQUESTED** [review]

**Comment by margaretkennedy** [comment]
https://github.com/deephaven/deephaven.io/issues/1716

File: .github/workflows/build-ci.yml (hunk 1 above)
Comment (JamesXNelson) [reply .github/workflows/build-ci.yml]: fixed

File: docker/registry/all-ai-base/gradle.properties [hunk 2]
```diff
@@ -0,0 +1,3 @@
+io.deephaven.project.ProjectType=DOCKER_REGISTRY
+deephaven.registry.imageName=ghcr.io/deephaven/all-ai-base:latest
+deephaven.registry.imageId=ghcr.io/deephaven/all-ai-base@sha256:6e6d0056d60391409800698bc211ad8b59c5a962c291902da5c93d00243a9a8f
```
Comment (devinrsmith) [inline docker/registry/all-ai-base/gradle.properties:3]: I think we'll need to bump this to the latest - and update the requirements.txt file.

**Review by devinrsmith — CHANGES_REQUESTED** [review]

File: docker/registry/all-ai-base/gradle.properties (hunk 2 above)
Comment (devinrsmith) [reply docker/registry/all-ai-base/gradle.properties:3]: You may also want to update the logic in `docker/registry/scripts/update-requirements.sh` to ensure this image gets regularly bumped

Comment (JamesXNelson) [reply docker/registry/all-ai-base/gradle.properties]: done.

**Review by devinrsmith — APPROVED** [review]

**Pull request merged**